docs/src/main/sphinx/connector/blackhole.rst

```diff
@@ -97,3 +97,15 @@ delay before consuming or producing a new page::
       page_processing_delay = '5s'
     );
 
+.. _blackhole-sql-support:
+
+SQL support
+-----------
+
+The connector provides access to data temporarily stored in the Black Hole
+storage engine. The connector supports the following features:
+
+* :doc:`/sql/insert`
```

Comment:
INSERT is allowed, but the data is not persisted. it must be called out.

also, there are other supported stataments: SELECT, CREATE TABLE, CTAS

Reply:
I will include the supported statements list.